src/docs/environment/index.mdx

```diff
@@ -89,23 +89,55 @@ computer is actually compiling Python!
 make setup-pyenv
 ```
 
+fish users will need to manually set some of the environment variables. This only needs to be done
+once.
+
+```shell {tabTitle: Fish}
+set -Ux PYENV_ROOT $HOME/.pyenv
+# fish >=3.2.0
+fish_add_path $PYENV_ROOT/bin
+# fish <3.2.0
+set -U fish_user_paths $PYENV_ROOT/bin $fish_user_paths
+```
+
```

Comment:
It's possible to insert lines equivalent to the interactively run commands into the 
startup script instead, but it's difficult to get this right. A common fish mistake is to append the 
pyenv directory without any regard for whether it's already present in `PATH` or not, causing 
it to explode in length based on how often the user spins up new shells. 

In comparison, the fish commands proposed in this PR only need to be run once, will properly append to the `PATH`, and therefore should not cause too much developer pain.